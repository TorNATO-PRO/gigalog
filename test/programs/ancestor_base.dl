parent(john, mary).
parent(mary, bob).

ancestor(X, Y) :- parent(X, Y).
