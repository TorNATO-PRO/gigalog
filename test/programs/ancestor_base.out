ancestor(john,mary)
ancestor(mary,bob)
parent(john,mary)
parent(mary,bob)
